mnd/mnd/models.py

```diff
@@ -96,6 +98,15 @@ def clean(self):
             self.email = self.email.lower()
 
 
+@receiver(user_email_updated)
+def update_carer_email(sender, user, **kwargs):
```

Comment:
When I updated the carers email, it appeared to go successfully, and the carer still appears to be able to see the patient. However, on the principal caregiver management page, it says that the carer hasn't been registered:

![image](https://github.com/eresearchqut/mnd/assets/17246877/4810e447-7b07-4cf5-bd08-5b14ac1ef7b7)

Reply:
Good pick up. Looks like there was yet another email field that needed to be updated 😬